metashare/api/sf_org_changes.py

```diff
@@ -30,39 +94,24 @@ def run_retrieve_task(user, scratch_org, project_path, desired_changes):
         }
     )
 
-    # Determine default package directory
-    # Use src for mdapi format,
-    # or the default package directory from sfdx-project.json for sfdx format
-    # (This chunk is copied from CumulusCI, where we need to refactor things
-    # so we can reuse it. Given that it's already tested there, I'm going to
-    # pragma: no cover some of it.)
-    package_directories = []
-    default_package_directory = None
-    package_xml_opts = {}
-    sfdx_project_json = os.path.join(project_path, "sfdx-project.json")
-    if os.path.exists(sfdx_project_json):
-        with open(sfdx_project_json, "r") as f:  # pragma: no cover
-            sfdx_project = json.load(f)
-            for package_directory in sfdx_project.get("packageDirectories", []):
-                package_directories.append(package_directory["path"])
-                if package_directory.get("default"):
-                    default_package_directory = package_directory["path"]
-    if (
-        default_package_directory
-        and cci.project_config.project__source_format == "sfdx"
-    ):  # pragma: no cover
-        path = os.path.join(project_path, default_package_directory)
-        md_format = False
+    valid_directories, sfdx = get_valid_target_directories(
+        user, scratch_org, project_path
+    )
+    md_format = target_directory not in valid_directories["source"]
```

Comment:
```suggestion
    md_format = not (sfdx and target_directory in valid_directories["source"])
```

@wlonk My understanding is that this should also take into account whether or not `sfdx` is truthy?